tests/line.c

```diff
@@ -150,70 +150,288 @@ void svpng(FILE *fp, unsigned w, unsigned h, const uint8_t *img, bool alpha)
     SVPNG_END(); /* IEND chunk {} */
 }
 
-#include <math.h>  // ceilf(), floorf(), fminf(), fmaxf(), sinf(), cosf(), sqrtf()
 #include <string.h>  // memset()
+#define Q (20)
+#define Q_PI (1686629713 >> (29 - Q))
+
+/* 32-bit Q notation to specify a fixed point number format. */
+typedef int32_t q_fmt;
```

Comment:
If you read [CONTRIBUTING.md](https://github.com/sysprog21/rv32emu/blob/master/CONTRIBUTING.md) carefully, you would be aware of suffix `_t` used in typedef'd types. Here, for the sake of consistency, we can rewrite as following:
```c
typedef int32_t qfixed_t;
typedef int64_t qbuf_t;
```